### DISCUSSION TITLE
Disable deleting a file in File Manager or Desktop by pressing DELETE key

### DISCUSSION THREAD

**Discussion by murty2** [DustinBrett/daedalOS#442] [Q&A]
If select an icon on Desktop or in File Manager, and then press DELETE key, the icon is deleted. How can I just ignore the DELETE key press and not delete it?

**Comment by DustinBrett** [comment]
In `components\system\Files\FileManager\useFileKeyboardShortcuts.ts`, handle `"Delete"` case differently.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/e79d5fca-ca32-4266-b4ce-b57ef412814c)
